README.md

```diff
@@ -91,7 +91,8 @@ Lint all `yaml` files in `path` and all subdirectories (recursive):
 
 - `cfn-lint path/**/*.yaml`
 
-*Note*: Glob in Python 3.5 supports recursive searching `**/*.yaml`.  If you are using an earlier version of Python you will have to handle this manually (`folder1/*.yaml`, `folder2/*.yaml`, etc).
+*Note*: If using sh/bash/zsh, you must enable globbing.
+(`setopt -s globstar` for sh/bash, `setopt extended_glob` for zsh).
```

Comment:
not sure about `sh`, but think it's `shopt` instead of `setopt` for `bash`:
https://www.gnu.org/software/bash/manual/html_node/The-Shopt-Builtin.html

Reply:
@kddejong Python 3.5 note is still relevant, right?
https://github.com/aws-cloudformation/cfn-python-lint/pull/549

Reply:
ah yes, this was an error in transcription. `shopt` for bash, `setopt` for zsh.

Reply:
@PatMyron unless there is a test case where this works *without* bash or zsh expansion, I would leave off the Python note. The Python note only conveys meaning if the expansion actually works in Python. Currently, I do not think it does (hence this PR).

Reply:
my `**` seems to work on my setup without that option set using zsh.  I can repeat this using bash so I could see the value of this change.